src/tlo/methods/healthsystem.py

```diff
@@ -1032,6 +1052,20 @@ def get_mode_appt_constraints(self) -> int:
         return self.parameters['mode_appt_constraints'] \
             if self.arg_mode_appt_constraints is None \
             else self.arg_mode_appt_constraints
+            
+    def get_year_mode_switch(self) -> int:
+        """Returns `year_mode_switch`. (Should be equal to what is specified by the parameter, but overwrite with
+        what was provided in argument if an argument was specified -- provided for backward compatibility/debugging.)"""
+        return self.parameters['year_mode_switch'] \
+            if self.arg_year_mode_switch is None \
+            else self.arg_year_mode_switch
+
+    def get_scale_to_effective_capabilities(self) -> int:
+        """Returns `scale_to_effective_capabilities`. (Should be equal to what is specified by the parameter, but overwrite with
+        what was provided in argument if an argument was specified -- provided for backward compatibility/debugging.)"""
+        return self.parameters['scale_to_effective_capabilities'] \
+            if self.arg_scale_to_effective_capabilities is None \
+            else self.arg_scale_to_effective_capabilities
```

Comment:
```suggestion
    @property
    def year_mode_switch(self) -> int:
        """Returns `year_mode_switch`. (Should be equal to what is specified by the parameter, but overwrite with
        what was provided in argument if an argument was specified -- provided for backward compatibility/debugging.)"""
        return (
            self.parameters['year_mode_switch']
            if self._arg_year_mode_switch is None
            else self._arg_year_mode_switch
        )

    @property
    def scale_to_effective_capabilities(self) -> int:
        """Returns `scale_to_effective_capabilities`. (Should be equal to what is specified by the parameter, but overwrite with
        what was provided in argument if an argument was specified -- provided for backward compatibility/debugging.)"""
        return (
            self.parameters['scale_to_effective_capabilities']
            if self._arg_scale_to_effective_capabilities is None
            else self._arg_scale_to_effective_capabilities
        )
```

Decorating these methods as properties allows accessing like standard attributes (for example `health_system.scale_to_effective_capabilities`) and is more 'Pythonic' than defining getter-methods. Also generally [explicit line continuation characters are discouraged in Python in favour of using implicit line continuation by wrapping in parenthesis](https://peps.python.org/pep-0008/#maximum-line-length).